qmlti/src/resources/LTI_Data_Connector_qmp.php

```diff
@@ -748,6 +748,66 @@ public function Results_getParticipantsByResource($consumer, $resource_link, $as
   }
 
 ###
+###  lTI_Attempts methods
+###
+
+/*
+ *  Gets the latest external attempt ID for an assessment
```

Comment:
Missing PHPDoc. Probably a good idea if someone else has to maintain this in the future.